ambry-frontend/src/main/java/com/github/ambry/frontend/AmbrySecurityService.java

```diff
@@ -51,7 +53,7 @@
  * sets the respective headers on response.
  */
 class AmbrySecurityService implements SecurityService {
-
+  private static final Logger logger = LoggerFactory.getLogger(AmbrySecurityService.class);
```

Comment:
logger is not used. Did you want to log anything?

Reply:
no, let me remove it.